docs: READMEとドキュメント類を追加 (#62)
tomacheese/ElitesRNGAuraObserver: docs/en/installation/configuration.md

# Application Settings

This page explains the configurable items in Elite's RNG Aura Observer and what you can check on the settings screen.

![Settings screen of Elite's RNG Aura Observer](/docs/assets/installation/settings-ui.png)

> To open the settings screen, double-click the application icon in the taskbar or right-click and select `Settings`.

Reviewer: 日本語版と同様、タスクバー→タスクトレイ

```suggestion
To open the settings screen, double-click the application icon in the task tray or right-click and select `Settings`.
```

## Monitored Log File Path

In `Monitoring Information` > `Monitoring Log File`, you can check the path of the log file used to detect whether an Aura has been obtained.

## Discord Webhook URL

By entering the Discord Webhook URL in `Discord Settings` > `Discord Webhook URL`, you can set the notification destination.

For how to obtain the Discord Webhook URL, please see the following page:

- [How to get Discord Webhook URL](get-discord-webhook-url.md)

## Enable Windows Toast Notification

> By checking the `Enable Toast notification` checkbox in `Application Settings`, you will receive a Windows toast notification when an Aura is obtained.

Reviewer: 日本語での指摘内容に合わせて

```suggestion
When the `Enable Toast Notification` checkbox in `Application Settings` is checked, Windows will display a toast notification upon acquiring Aura.
```
